manager/src/main/java/io/gingersnapproject/Caches.java

```diff
@@ -61,6 +71,16 @@ public Uni<String> put(String name, String key, String value) {
       return indexUni;
    }
 
+   public Uni<List<String>> putAll(String name, Map<String, String> values) {
```

Comment:
This should be in a separate PR imo. We need to make sure to have getAll in first.